packages/@storefront/flux-capacitor/src/core/store/middleware.ts

```diff
@@ -175,6 +190,7 @@ export namespace Middleware {
     ];
     const middleware = [
       ...normalizingMiddleware,
+      Middleware.updateHistory(flux),
       Middleware.saveStateAnalyzer(),
       Middleware.injectStateIntoRehydrate,
       Middleware.validator,
```

Comment:
If it's related and it directly affects the change, do it here. Otherwise, do it separately so we can revert it separately if we need to. Maybe switch it first so it gets absorbed in the major release.

Reply:
Sure makes sense. I don't think it's directly related I just noticed it and was like 🤦‍♀️